### PR TITLE
Update to Angular 16

### PR DESCRIPTION
**Pull request by fjaguado**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,9 +52,9 @@
     "karma-coverage-istanbul-reporter": "~2.1.0",
     "karma-jasmine": "~2.0.1",
     "karma-jasmine-html-reporter": "1.5.4",
-    "ng-packagr": "^15.0.3",
+    "ng-packagr": "16.2.3",
     "protractor": "~7.0.0",
```
Comment (joaquimvila) [inline package.json:56]: Are we using protractor in this library? If that's not the case, I would get rid of this dependency and if later a test automation project is created, then we should add WDIO.

**Review by joaquimvila — CHANGES_REQUESTED** [review]
I miss updating the package.json file under "projects/systelab-translate" folder.
Notice the range of accepted Angular versions should be updated there.

File: package.json (hunk 1 above)
Comment (fjaguado) [reply package.json]: No, we don't. This has been removed

**Review by oGalceran1 — APPROVED** [review]

**Review by jserra91 — APPROVED** [review]

File: projects/systelab-translate/package.json [hunk 2]
```diff
@@ -19,10 +19,10 @@
...
   "private": false,
   "peerDependencies": {
-    "@angular/common": ">=15.0.0 <16.0.0",
-    "@angular/core": ">=15.0.0 <16.0.0",
-    "@ngx-translate/core": ">=14.0.0 <15.0.0",
-    "@ngx-translate/http-loader": ">=7.0.0 <8.0.0",
-    "date-fns": ">=2.0.0 <3.0.0"
+    "@angular/common": "^16.0.0",
+    "@angular/core": "^16.0.0",
+    "@ngx-translate/core": "^14.0.0",
```
Comment (joaquimvila) [inline projects/systelab-translate/package.json:24]: According to docs, this one should be moved to "^15.0.0": https://github.com/ngx-translate/core

Comment (fjaguado) [reply projects/systelab-translate/package.json:24]: Done

**Review by joaquimvila — APPROVED** [review]

**Review by RubKrmon — APPROVED** [review]

**Pull request merged**